readit/__init__.py

```diff
@@ -1,3 +1,20 @@
+'''
+ Copyright (C) 2017, 2018 projectreadit organization and contributors.
+ This file is part of Readit - Command Line Bookmark Manager Tool.
+
+ This project is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This project is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with readit.  If not, see <http://www.gnu.org/licenses/>.
+'''
 from readit.cli import main
```

Comment:
One blank line after this text would look cleaner I think.

Reply:
@bhavin192 
Yes, I have applied it to all three files. Please take a look at it and approve this PR.